docs/qiita-16S-analysis.rst

```diff
@@ -3,218 +3,424 @@
+The "Background Color" pull-down menu allows you to change the color of the background of the PCoA.
+The % Variantion Expanded graph displays how different the most dissimilar samples are by percentage for each axis that can be used. 
+
+Under the "Scale" tab you will notice 2 pull-down menus:
+
+.. figure::  images/scale_tab.png
+   :align:   center
+
+Under "Select a Scale Category" you can choose the grouping of your samples. Under "Global Scaling" you can change the point size for each group on the PCoA plot. 
+
+Here, I’ve colored the points in our test data by cat ownership.
+
+.. figure::  images/pcoa.png
+   :align:   center
+
+Let’s take a few minutes now to explore the various features of Emperor. Open a new browser window with the 'Emperor tutorial <https://biocore.github.io/emperor/tutorial_index.html#section1>'__ and follow along with your test data.
+
+Another way to study the beta diversity is by measuring the beta diversity group significance. Beta diversity group significance measures whether groups of samples are significantly different from one another using a permutation-based statistical test.
+
+To perform a beta group significance analysis, select the rarefied "dflt_name - BIOM" artifact in the processing network and select "Process". Then select the "dflt_name - BIOM" artifact and select "beta_diversity" from the drop-down menu. The parameters will appear below the workflow diagram: 
+
+.. figure::  images/beta_group_significance_beta.png
+   :align:   center
+
+Several parameter have automatically selected for you. In the "p-metric" field enter "unweighted Unifrac" and in the "i-tree" field enter "/databases/gg/13_8/trees/97_otus.tree", and click "Add Command".
+
+To create the beta group significance analysis, select select "beta_group_significance" from the drop-down menu. The parameters will appear below the workflow diagram:
+
+.. figure::  images/significance_matrix.png
+   :align:   center
+
+All of the parameter have automatically selected for you just click "Add Command". Once the command is added the workflow should appear as follows:
+
+.. figure::  images/beta_group_significance_workflow.png
+   :align:   center
+
+Once the q2 visualization artifact is chosen in the network, the beta diversity box plots will appear: 
+
+.. figure::  images/beta_significance_boxplot.png
+   :align:   center
+
+The PERMANOVA (Permutational multivariate analysis of variance) results will also be displayed:
+
+.. figure::  images/permanova_results.png
+   :align:   center
+
+Altering Workflow Analysis Names
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To alter the name of a result, click the artifact then use the edit button on the processing network page. 
+
+.. figure::  images/rename_data_on_workflow.png
+   :align:   center
+
+This will cause a window to pop-up where you can input the name you’d like to replace it with.
+
+.. figure::  images/rename_data_popup.png
+   :align:   center
+
+Analysis of Deblur Process
```

Comment:
I'm wondering if we should keep deblur here with examples, or just drop it altogether. I can generate deblur BIOM, but maybe we can just leave it as a description for curious people, without actual example data?
@adswafford what do you think?